wiki/Gameplay/Game_modifier/Mod_multiplier/en.md

```diff
@@ -1,27 +1,28 @@
 ---
-outdated: true
 stub: true
 tags:
   - score multiplier
   - score
+  - mod
+  - mod multiplier
 ---
 
-<!--Outdated because the math and formula to the score multiplier is wrong. Spawned from issue #4448-->
-
-# Score multiplier
+# Mod multiplier
 
+*Not to be confused with: [Combo](/wiki/Gameplay/Combo_(score_multiplier))*\
 *See also: [Game modifier](/wiki/Game_modifier)*
 
-<!-- TBA: Not to be confused with: [Combo](link) -->
-
 ## Description
 
-The **Score Multiplier** is a value used to determine how much osu! should multiply the [score](/wiki/Gameplay/Score) received from each individual [hit object](/wiki/Hit_object) throughout a play. By default, the Score Multiplier will be 1.00x, but this value can be changed through the use of [game modifiers](/wiki/Game_modifier).
+The **Mod Multiplier** is a value used to calculate the [score](/wiki/Gameplay/Score) throughout a play depending on selected [game modifiers](/wiki/Game_modifier). By default, the Mod Multiplier will be 1.00x.
+
+<!--What for the effects of different mods (and their combinations) on this multiplier?-->
```

Comment:
in my opinion, this comment is unnecessary, as it's described in the last paragraph of this section: "However, if multiple mods are enabled, ..."
```suggestion
```